trojsten/rules/susi.py

```diff
@@ -112,11 +110,15 @@ def prepare_coefficients(self, round):
                     task__round__semester__number__lt=round.semester.number,
                 ),
             )
-            .values("user")
+            .values("user", "task__round__semester")
             .annotate(solved_tasks=Count("task", distinct=True))
             .values_list("user", "solved_tasks")
         )
 
+        for user_id, points in no_of_solved_tasks_per_semester:
```

Comment:
points by som premenoval, pretože názov je zmätočný, je to počet submitov/úloh, nie bodov.

Reply:
ok, zmenil som